cql3/statements/create_view_statement.cc

```diff
@@ -13,6 +13,12 @@
 #include <unordered_set>
 #include <vector>
 
+#include <boost/regex.hpp>
+#include <boost/range/iterator_range.hpp>
+#include <boost/range/join.hpp>
+#include <boost/range/adaptor/map.hpp>
+#include <boost/range/adaptor/transformed.hpp>
```

Comment:
I don't think you used any of these boost range header files, so please see which of them you can remove. I think you only really used regex.

Reply:
done